Tasks/DownloadPackageV0/task.json

```diff
@@ -9,13 +9,12 @@
   "author": "ms-vscs-rm",
   "version": {
     "Major": 0,
-    "Minor": 231,
+    "Minor": 232,
     "Patch": 0
   },
   "demands": [],
   "minimumAgentVersion": "2.144.0",
   "deprecated": true,
-  "removalDate": "2023-11-27",
```

Comment:
This conflicts with [this change](https://github.com/microsoft/azure-pipelines-tasks/pull/19325/files#diff-6fbd30badefb60031b67a771be66616834dbd4d8b11ff54bbc07732382aedba6) in #19325